src/NUnitFramework/tests/Attributes/TestCaseSourceTests.cs

```diff
@@ -87,72 +84,28 @@ public void SourceCanBeStaticField(string source)
         static object[] StaticField =
             { new object[] { "StaticField" } };
 
-        [Test, TestCaseSource("InstanceField")]
-        public void SourceCanBeInstanceField(string source)
+        [Test]
+        public void SourceUsingInstanceFieldIsNotRunnable()
```

Comment:
I may have missed them, but should we also have the Static...IsRunnable tests for this and ValueSource?